Remotion/ObjectBinding/Web/UI/Controls/BocBooleanValueImplementation/Rendering/BocCheckBoxRenderer.cs

```diff
@@ -257,8 +261,13 @@ private string GetDescription (BocCheckBoxRenderingContext renderingContext)
         string defaultTrueDescription = renderingContext.Control.DefaultTrueDescription;
         string defaultFalseDescription = renderingContext.Control.DefaultFalseDescription;
```

Comment:
Please update the type for the Default... values, too (consistency). in the end, we can return WebString instead of string.